### PR TITLE
installation: invenio.config reference fixes

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes references to invenio.config and replaces them with
  references to invenio_base.globals.cfg.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

File: docs/getting-started/overlay.rst [hunk 1]
```diff
@@ -131,7 +131,7 @@ it later on. Here is its minimal content.
             "Invenio>=2"
         ],
         entry_points={
-            "invenio.config": ["myoverlay = myoverlay.config"]
+            "invenio_base.globals.cfg": ["myoverlay = myoverlay.config"]
```
Comment (jirikuncar) [inline docs/getting-started/overlay.rst]: this one should not be replaced

File: docs/admin/webstyle-admin-guide.rst [hunk 2]
```diff
@@ -163,13 +163,13 @@ named ``webstyle_templates_ithaca.py`` with the following content:
...
 
-        from invenio.config import CFG_SITE_LANG
+        from invenio_base.globals import cfg
         from invenio.legacy.webstyle.templates import Template as DefaultTemplate
 
         class Template(DefaultTemplate):
             """Ithaca style templates."""
 
-            def tmpl_pagefooter(self, req=None, ln=CFG_SITE_LANG, lastupdated=None,
+            def tmpl_pagefooter(self, req=None, ln=cfg['CFG_SITE_LANG'], lastupdated=None,
```
Comment (jirikuncar) [inline docs/admin/webstyle-admin-guide.rst]: the whole file should be removed together with other legacy admin guides

**Comment by jirikuncar** [comment]
@SamiHiltunen we should instead remove the rest of legacy except `run_sql` unless `invenio-access` is amended first.

**Comment by SamiHiltunen** [comment]
@jirikuncar I'll start working on access in this case

**Pull request closed without merging**